recipe-server/normandy/recipes/api/views.py

```diff
@@ -138,6 +160,32 @@ class RecipeRevisionViewSet(viewsets.ReadOnlyModelViewSet):
     ]
 
 
+class ApprovalRequestViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = ApprovalRequest.objects.all()
+    serializer_class = ApprovalRequestSerializer
+    permission_classes = [
+        AdminEnabledOrReadOnly,
+    ]
+
+    @detail_route(methods=['POST'])
+    def approve(self, request, pk=None):
+        approval_request = self.get_object()
+        approval_request.approve(approver=request.user)
+        return Response(ApprovalRequestSerializer(approval_request).data)
+
+    @detail_route(methods=['POST'])
+    def reject(self, request, pk=None):
+        approval_request = self.get_object()
+        approval_request.reject(approver=request.user)
+        return Response(ApprovalRequestSerializer(approval_request).data)
+
+    @detail_route(methods=['POST'])
+    def close(self, request, pk=None):
+        approval_request = self.get_object()
+        approval_request.delete()
+        return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
I thought that we talked about closing an approval also removing the changes in the approval? Or was that rejecting? Or did we drop that?